consensus/service/src/config/tokens.rs

```diff
@@ -25,6 +82,14 @@ pub struct TokenConfig {
     // file.
     #[serde(default)]
     allow_any_fee: bool,
+
+    /// Master minters - if set, controls the set of keys that can sign
+    /// set-minting-configuration transactions.
+    /// Not supported for MOB
+    #[serde(serialize_with = "signer_set_ser")]
+    #[serde(deserialize_with = "signer_set_deser")]
```

Comment:
Optional/Equivalent: [`serde(with="module")` syntax](https://serde.rs/field-attrs.html#with). Alternatively, we could declare these associated types at the `SignerSet` level, with `serde(into="DerSignerSet", try_from="DerSignerSet")`

Reply:
I didn't want to do it directly on the `SingerSet` since I don't want to go through DER and hex encoding when going in and out of the enclave

Reply:
The `with` syntax is nice, I'll try that